.erb-lint.yml

```
@@ -151,7 +151,7 @@ linters:
           - 'btn(-(small|big|narrow|wide|link|primary|secondary|danger|disabled|big|narrow|transparent|border))?'
           - '(border|bg)-(none|black|gray|silver|aqua|blue|navy|teal|green|olive|lime|orange|red|fuchsia|purple|maroon|darken-[1-4]|lighten-[1-4])'
           - 'bg-(cover|contain|center|top|right|bottom|left)'
-          - 'h([03]|00)'
+          - 'h([013]|00)'
```

should we do a range?
```suggestion
          - 'h([0-6]|00)'
```

---

> should we do a range?

Eventually, yes. We've still got some work to do to excise the rest of them, though:

```
Deprecated class `h5` detected matching the pattern `h([0-6]|00)`. Use USWDS classes instead of BassCSS.
In file: app/views/devise/shared/_password_strength.html.erb:9

Deprecated class `h6` detected matching the pattern `h([0-6]|00)`. Use USWDS classes instead of BassCSS.
In file: app/views/devise/shared/_password_strength.html.erb:10

Deprecated class `h6` detected matching the pattern `h([0-6]|00)`. Use USWDS classes instead of BassCSS.
In file: app/views/devise/shared/_password_strength.html.erb:17

Deprecated class `h2` detected matching the pattern `h([0-6]|00)`. Use USWDS classes instead of BassCSS.
In file: app/views/idv/doc_auth/welcome.html.erb:18

Deprecated class `h2` detected matching the pattern `h([0-6]|00)`. Use USWDS classes instead of BassCSS.
In file: app/views/idv/doc_auth/welcome.html.erb:33

Deprecated class `h2` detected matching the pattern `h([0-6]|00)`. Use USWDS classes instead of BassCSS.
In file: app/views/idv/doc_auth/welcome.html.erb:49

Deprecated class `h2` detected matching the pattern `h([0-6]|00)`. Use USWDS classes instead of BassCSS.
In file: app/views/idv/doc_auth/welcome.html.erb:65

Deprecated class `h2` detected matching the pattern `h([0-6]|00)`. Use USWDS classes instead of BassCSS.
In file: app/views/idv/doc_auth/link_sent.html.erb:22

Deprecated class `h6` detected matching the pattern `h([0-6]|00)`. Use USWDS classes instead of BassCSS.
In file: app/views/shared/_pii_review.html.erb:2

Deprecated class `h4` detected matching the pattern `h([0-6]|00)`. Use USWDS classes instead of BassCSS.
In file: app/views/shared/_pii_review.html.erb:6

Deprecated class `h6` detected matching the pattern `h([0-6]|00)`. Use USWDS classes instead of BassCSS.
In file: app/views/shared/_pii_review.html.erb:10

Deprecated class `h4` detected matching the pattern `h([0-6]|00)`. Use USWDS classes instead of BassCSS.
In file: app/views/shared/_pii_review.html.erb:14

Deprecated class `h6` detected matching the pattern `h([0-6]|00)`. Use USWDS classes instead of BassCSS.
In file: app/views/shared/_pii_review.html.erb:18

Deprecated class `h4` detected matching the pattern `h([0-6]|00)`. Use USWDS classes instead of BassCSS.
In file: app/views/shared/_pii_review.html.erb:22

Deprecated class `h6` detected matching the pattern `h([0-6]|00)`. Use USWDS classes instead of BassCSS.
In file: app/views/shared/_pii_review.html.erb:26

Deprecated class `h4` detected matching the pattern `h([0-6]|00)`. Use USWDS classes instead of BassCSS.
In file: app/views/shared/_pii_review.html.erb:30

Deprecated class `h6` detected matching the pattern `h([0-6]|00)`. Use USWDS classes instead of BassCSS.
In file: app/views/shared/_pii_review.html.erb:35

Deprecated class `h4` detected matching the pattern `h([0-6]|00)`. Use USWDS classes instead of BassCSS.
In file: app/views/shared/_pii_review.html.erb:38

Deprecated class `h5` detected matching the pattern `h([0-6]|00)`. Use USWDS classes instead of BassCSS.
In file: app/views/shared/_footer_lite.html.erb:9

Deprecated class `h4` detected matching the pattern `h([0-6]|00)`. Use USWDS classes instead of BassCSS.
In file: app/views/shared/_failure.html.erb:18

Deprecated class `h2` detected matching the pattern `h([0-6]|00)`. Use USWDS classes instead of BassCSS.
In file: app/views/accounts/show.html.erb:34

Deprecated class `h2` detected matching the pattern `h([0-6]|00)`. Use USWDS classes instead of BassCSS.
In file: app/views/accounts/_emails.html.erb:3

Deprecated class `h4` detected matching the pattern `h([0-6]|00)`. Use USWDS classes instead of BassCSS.
In file: app/views/users/edit_phone/edit.html.erb:10

Deprecated class `h2` detected matching the pattern `h([0-6]|00)`. Use USWDS classes instead of BassCSS.
In file: app/views/users/backup_code_setup/create.html.erb:23

Deprecated class `h4` detected matching the pattern `h([0-6]|00)`. Use USWDS classes instead of BassCSS.
In file: app/views/users/webauthn_setup/new.html.erb:50

Deprecated class `h4` detected matching the pattern `h([0-6]|00)`. Use USWDS classes instead of BassCSS.
In file: app/views/reactivate_account/index.html.erb:22

Deprecated class `h5` detected matching the pattern `h([0-6]|00)`. Use USWDS classes instead of BassCSS.
In file: app/views/partials/personal_key/_key.html.erb:15

Deprecated class `h2` detected matching the pattern `h([0-6]|00)`. Use USWDS classes instead of BassCSS.
In file: app/views/two_factor_authentication/webauthn_verification/show.html.erb:31

Deprecated class `h4` detected matching the pattern `h([0-6]|00)`. Use USWDS classes instead of BassCSS.
In file: app/views/two_factor_authentication/webauthn_verification/show.html.erb:44
```

---

got it! 👍